File: Java/src/Raoni/Act01/Principal.java
package Raoni.Act01;

public class Principal {
    public static void main(String[] args) {
        Aluno a1 = new Aluno("Francisco","Masculino",1235854,11323255,58154,true);
        Professor p1 = new Professor("josivaldo","Masculino","novidade do amanhã",12521598,02551);
        Professor p2 = new Professor("Roberta","Feminino","Resistência",65818518,12232);
        Professor p3 = new Professor("João","Masculino","Resistência",265515,20202);
        Professor p4 = new Professor("Ana","Feminino","Revolução",25845111,15555);

        Professor[] professores = {p1 ,p2 ,p3 ,p4};

        Turma t1 = new Turma(04,12,"Física",2,true,professores);
        Turma t2 = new Turma(25,02,"Matemática",2,true,professores);
        System.out.println(t1.toString());
        System.out.println("_____________________Elenco de professores____________________");
        t1.imprimirProfessores();

        Turma[] turmas = {t1, t2};

        p1.setTurmas(turmas);

        p1.imprimirTurmas();
    }
}

File: C#/ConsoleApplication3/ConsoleApplication3/Atividades_Raoni/atividade_01/Principal.cs
using System;

namespace ConsoleApplication01.Atividades_Raoni
{
    public class Principal
    {
        public static void Maimn(string[] args)
        {
            Aluno a1 = new Aluno("Francisco","Masculino",12581,022515,02222,"ADS");
            Professor p1 = new Professor("Jose", "Masculino", "Assas livres", 28415655, 02020, "Física");
            Professor p2 = new Professor("Maria", "Feminino", "Novidade nova", 12551, 02025, "Matemática");
            Professor p3 = new Professor("Claúdia", "Feminino", "Assas livres", 251455, 02520, "Literatura");
            Professor p4 = new Professor("Gustavo", "Masculino", "Revolução", 2584884, 15515, "Geografia");

            Turma t1 = new Turma(12510, 02, "Física", "02");
            Professor[] professores = {p1, p2, p3, p4};
            t1.professores = professores;
            
            t1.imprimirProfessores();
        }
    }
}